src/common/components/help/help-install-snap.js

```diff
@@ -21,11 +21,17 @@ export default class HelpInstallSnap extends Component {
       <div className={styles.helpFlexWrapper}>
         <HeadingThree className={styles.helpFlexHeading}>{ headline }</HeadingThree>
         <div className={styles.helpText}>
-          <pre>
-            <code className={ styles.cli }>
+          <pre className={styles.cli}>
```

Comment:
Possibly related, going to such page throws a bunch of React warnings:
<img width="654" alt="screen shot 2017-08-17 at 09 04 00" src="https://user-images.githubusercontent.com/83575/29399833-189acc02-832b-11e7-8240-90adaf4fec94.png">